app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -92,6 +94,46 @@ class WalkthroughTopicListFragmentTest {
     )
   }
 
+  @Test
+  fun testWalkthroughWelcomeFragment_recyclerViewIndex1_topicHeader_notOnTop_progressBarIsHidden() {
+    launch<WalkthroughActivity>(createWalkthroughActivityIntent(0)).use {
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.walkthrough_welcome_next_button)).perform(scrollTo(), click())
+      onView(withId(R.id.walkthrough_topic_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          4
+        )
+      )
+      onView(withId(R.id.walkthrough_progress_bar)).check(
+        matches(
+          withEffectiveVisibility(
+            ViewMatchers.Visibility.GONE
+          )
+        )
+      )
+    }
+  }
+
+  @Test
+  fun testWalkthroughWelcomeFragment_recyclerViewIndex1_topicHeader_notOnTop_activityHeaderIsVisible() { // ktlint-disable max-line-length
```

Comment:
try to avoid using `// ktlint-disable max-line-length` by using short or abbreviated test function name.

Reply:
Done